backend/VoterRegistrationService/README.md

```diff
@@ -0,0 +1,103 @@
+# AzureFunction
+Initial setup for AccessAmerica Voter Registration 
```

Comment:
So after the conversation with @sarah-powell, looks like we're getting rid of specific state backend validations.
The request will be generic for all states, I have to rip out all those state specific validations

Reply:
I'll then include instructions on how to run this in VS Code in this README
I've been working in Visual Studio directly because I'm just more used to it but anyone should be able to run this locally using VS Code (windows or mac)

Reply:
I think you’re referring to those specific fields that added additional validation, like “have you convicted a felony” and stuff like that. Is that correct? There will still be some basic validation regarding which field are required for which states.

Reply:
Exactly that @sarah-powell. From what I understood, those specific requirements will be presented to the user some how on the front-end and user will have to acknowledge those terms.
The request to the validation service won't need to include those specific fields anymore, just the generic ones like name / address / date of birth...

Reply:
@ricky-sb updated the README file and added valid examples on how the request body should look.
Removed all specific states requirement validations so regardless of state, request will always be identical. 
If you want to run this locally (windows or mac), you can do so using VS Code